docs/HyperIndex/Guides/navigating-hasura.md

```diff
@@ -138,6 +138,66 @@ When testing, you may need to reset your database:
 - **Schema Validation**: Use the Data tab to verify that relationships between entities are correctly established
 - **Performance Monitoring**: Watch for tables that grow unusually large, which might indicate inefficient indexing
 
+## Aggregations: local vs hosted (avoid the foot‑gun)
+
+When developing locally with Hasura, you may notice that GraphQL aggregate helpers (for example, count/sum-style aggregations) are available. On the hosted service, these aggregate endpoints are intentionally not exposed. Aggregations over large datasets can be very slow and unpredictable in production.
+
+The recommended approach is to compute and store aggregates at indexing time, not at query time. In practice this means maintaining counters, sums, and other rollups in entities as part of your event handlers, and then querying those precomputed values.
+
+### Example: indexing-time aggregation
+
+schema.graphql
+
+```graphql
+# singleton; you hardcode the id and load it in and out
+entity GlobalState {
+  id: ID! # "global-state"
+  count: INT!
+}
+
+entity Token {
+  id: ID! # incremental number
+  description: String!
+}
+```
+
+EventHandler.ts
+
+```typescript
+const globalStateId = "global-state";
+
+NftContract.Mint.loader((event, context) => {
+  context.GlobalState.load(globalStateId);
+});
```

Comment:
@DenhamPreen This is invalid API

There's also missing `await` in front of `context.GlobalState.get` and Handler args are invalid